sql/mob_droplist.sql

```diff
@@ -17416,29 +17416,31 @@ INSERT INTO `mob_droplist` VALUES (3222,2,0,1000,1455,0); -- one_byne_bill
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11295,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11298,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11307,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11820,@VCOMMON); -- khthonios_mask
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15088,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15089,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15090,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15091,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15094,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15096,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15098,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15099,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15100,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15101,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15117,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15122,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15123,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15140,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15142,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,16360,@VRARE);
-INSERT INTO `mob_droplist` VALUES (3223,0,0,1000,16017,@COMMON);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,3493,50); -- forgotten_thought
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11292,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11295,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11298,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11307,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,11820,240); -- khthonios_mask
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15088,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15089,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15090,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15091,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15094,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15096,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15098,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15099,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15100,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15101,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15117,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15122,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15123,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15140,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,15142,10);
+INSERT INTO `mob_droplist` VALUES (3222,0,0,1000,16360,10);
+INSERT INTO `mob_droplist` VALUES (3223,0,0,1000,16017,150);
+INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,1604,100); -- Mannequin Legs
+INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,1605,100); -- Mannequin Feet
```

Comment:
I chewwed up these db changes during my rebasing. They'll need to adopt the new `@COMMON` syntax. Also, these could be PR'd to `base` ahead of time so that they don't conflict with any other db changes.